smatch/tests/test_matcher.py

```diff
@@ -106,6 +119,61 @@ def test_matcher_radius():
         assert set(idxc) == set(idx[ic])
 
 
+def test_matcher_radius_indices():
+    ra, dec = _gen_sphere_pts(50, 4543)
+    mch = Matcher(ra, dec)
+
+    rap = ra[::-1] + 0.1
+    decp = dec[::-1] + 0.1
+
+    idx, i1, i2, d = mch.query_radius(rap, decp, 0.2, return_indices=True)
+
+    assert np.max(d) < 0.2
+    dist_match = sphdist(ra[i1], dec[i1], rap[i2], decp[i2])
+    assert np.max(dist_match) < 0.2
+
+
+def test_matcher_radius_indices():
```

Comment:
Why is this test repeated?

Reply:
Because it's such a damn important test!

Or because of a cut-and-paste error.